sqlalchemy_utils/functions/database.py

```diff
@@ -420,10 +420,13 @@ def is_auto_assigned_date_column(column):
     )
 
 
-def database_exists(url):
+def database_exists(url, postgres_db=['postgres', 'template0', 'template1',
```

Comment:
Note: default params should never be mutable objects in Python. It's a common gotcha: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
So it would be better to make it a tuple? Or `None`?

Reply:
Best to default to None and then set to the current default list inside the function if the param is None